web/src/Web.App/Views/TrustComparators/Index.cshtml

```diff
@@ -58,4 +61,10 @@
     className = "govuk-grid-column-full"
 })
 
-<div id="compare-your-trust" data-id="@Model.CompanyNumber"></div>
\ No newline at end of file
+<feature name="@FeatureFlags.HighExecutivePay">
```

Comment:
It might be clearer to resolve this in the view model and set `showHighExec = Model.NewProp.ToString().ToLower()` rather than use a `<feature> tag